index.html

```diff
@@ -0,0 +1,781 @@
+            </li>
+            <li class="inline-footer ">
+
+                <ul>
+                    <li class="lh">MORE</li>
+                    <li><a href=" ">Reader Center</a></li>
+                    <li><a href=" ">Wirecutter</a></li>
+                    <li><a href=" ">Live Events</a></li>
+                    <li><a href=" ">The Learning Network</a></li>
+                    <li><a href=" ">Tools & Services</a></li>
+                    <li><a href=" ">Multimedia</a></li>
+                    <li><a href=" ">Photography</a></li>
+                    <li><a href=" ">Video</a></li>
+                    <li><a href=" ">Newsletters</a></li>
+                    <li><a href=" ">TimesMachine</a></li>
+                    <li><a href=" ">NYT Store</a></li>
+                    <li><a href=" ">Times Journey</a></li>
+                    <li><a href=" ">Manage My Account</a></li>
+                </ul>
+            </li>
+            <li class="inline-footer ">
+
+                <ul>
+                    <li class="lh">SUBSCRIBE</li>
+                    <li class="footer-bold"><a href=" ">Home Delivery</a></li>
+                    <li class="footer-bold"><a href=" ">Digital Subscriptions</a></li>
+                    <li class="footer-bold"><a href=" ">Games</a></li>
+                    <li class="footer-bold"><a href=" ">Cooking</a></li>
+                    <li class="very-small-text"><a href=" ">Email Newsletters</a></li>
+                    <li class="very-small-text"><a href=" ">Corporate Subscriptions</a></li>
+                    <li class="very-small-text "><a href=" ">Education Rate</a></li>
+
+                    <li class="very-small-text border-top"><a href=" ">Mobile Applications</a></li>
+                    <li class="very-small-text"><a href=" ">Replica Edition</a></li>
+                    <li class="very-small-text"><a href=" ">Español</a></li>
+                </ul>
+            </li>
+        </ul>
+        <div class="footer-bottom-links">
+
+            <ul class="border-top">
+                <li><a href=" ">&#169; 2020 The New York Times Company</a></li>
+                <li><a href=" ">NYTCo</a></li>
+                <li><a href=" ">Contact Us</a></li>
+                <li><a href=" ">Work with us</a></li>
+                <li><a href=" ">Advertise</a></li>
+                <li><a href=" ">T Brand Studio</a></li>
+                <li><a href=" ">Your Ad Choices</a></li>
+                <li><a href=" ">Privacy</a></li>
+                <li><a href=" ">Terms of Service</a></li>
+                <li><a href=" ">Terms of Sale</a></li>
+                <li><a href=" ">Site Map</a></li>
+                <li><a href=" ">Help</a></li>
+                <li><a href=" ">Subscriptions</a></li>
+            </ul>
+        </div>
+        <div>
+
+        </div>
+
+        <div class="footer-bottom">
+
+            <a href="# ">Access more of The Times by creating a free account or logging in.</a>
+        </div>
```

Comment:
- [ ]  The expandable footer should cover the full width and it also causes a horizontal scroll. Please make it covers the full with to create the same UI with the original page.

![image](https://user-images.githubusercontent.com/60184961/91615368-137eed00-e98c-11ea-88eb-6d8253dd08cd.png)